aspnetcore/host-and-deploy/health-checks/samples/2.x/HealthChecks.SqlServer/DependencyInjection/SqlServerHealthCheckBuilderExtensions.cs

```diff
@@ -0,0 +1,33 @@
+﻿using HealthChecks.SqlServer;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using System.Collections.Generic;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    public static class SqlServerHealthCheckBuilderExtensions
```

Comment:
I think it's worth having a section on writing a health check that's distributed as a library. The stuff thats in this class is basically our guidance. 

1. Write a health check as a standalone class - you can rely on DI, type activation, or named options to access configuration data
2. Write an extension method with parameters like the below to wire it up

Reply:
Sorry ... I should've explained ... this is only here because I don't have a BeatPulse package. I took their code and just jammed it into the sample temporarily. None of the code under `HealthChecks.SqlServer` will be in the final sample ... just a package reference to BeatPulse.